yogstation/code/modules/jungleland/jungle_megafauna.dm

```diff
@@ -133,8 +134,6 @@
 				rune_attack_chain()
 			if(TELEPORT_ATTACK)
 				teleport_attack_chain()
-			if(SPAWN_ATTACK)
```

Comment:
doesnt actually remove the existence of SPAWN_ATTACK so it will still include it in it's combo and stand there like an idiot